### ISSUE TITLE
Server when started not showing Address, etc

### ISSUE THREAD

**Issue by CJH500** [jweigelt/swbf2admin#24]
So when I start my game server, non of the info under the dashboard appears. The server is not displayed in game, but is running and I have port forwarded the game port both allowing the port in windows and on my router. I'm at a loss for figuring out why its not connecting.
![image](https://user-images.githubusercontent.com/55010720/64469537-c2973880-d12b-11e9-8dfb-0f5b6a66c288.png)
![image](https://user-images.githubusercontent.com/55010720/64469545-e0fd3400-d12b-11e9-9712-41f2d8e3f59f.png)

**Comment by jweigelt** [comment]
Live status requires runtime management to be enabled (see https://github.com/jweigelt/swbf2admin#optional-enabling-runtime-managament). When using runtime management with galaxy, set game- & rconport to the same value.

Possible reasons for the server not showing up are:
- galaxy communications service is stopped or crashing
- the server is hidden when using the same galaxy account for playing as for hosting

The Galaxy server will use the "game port" for rcon only. It will bind to a "random" port in the high range. Generally this port does not have to be forwarded. The rcon/gameport should **not** be open to WAN to not expose the server administrative console!

This does not apply to the gamespy variant of the server.

**Comment by CJH500** [comment]
Thank you, for your suggestions, enabling runtime management sorted the first issue.

As for seeing the server still no dice. So I am using a Steam version to play, that was not an issue, what is is the galaxy communication service. I have tried to get it to start but it won't at all, I have reinstalled galaxy also to no difference. Is Thier any know issue which prevents it from starting.

**Comment by jweigelt** [comment]
Alright so apparently the service is only started during server startup. It might not run while the server is running. You should see it starting briefly while the gameserver is launching.
You can also run the game itself to check if it connects to MP.

A known problem is the service crashing because it is incompatible with Windows Remote Desktop, so if you're managing the server using Windows RD you have to use something else.
The service crashing while launching the server causes the exact behaviour you're observing.

Another explanation would be udp packets being actively dropped. Standard firewall configuration should normally let them pass.

**Comment by CJH500** [comment]
So the standard multiplayer appears to work. I started the game and could see the server lost just fine. Still the server starts runs, but is not present in the server list/or given a IP in the management console. I did try prior to opening this issue completely disabling windows firewall and that seemed to make no difference at the time. I have hosted even if just for a day small servers in the past (half life 2, cod) so I’m unsure what’s going on, I know you mentioned about udp. As far as I can see nothing should be blocking packets.
